blog/_posts/2016-05-17-automated-data-validation.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Automated Data Validation with Data Packages
+author: Dan Fowler
+username: dfowler
+projects: [dpm,frictionless-data,mira,goodtables-web]
+---
+
+Software projects have long benefited from
```

Comment:
Suggest no links in the first paragraph. Info about understanding the background (if needed) can go at the end.

Why? Links suggest you need to follow out and distracting in first paragraph. In general, we can assume people know or can google.